.circleci/config.yml

```diff
@@ -22,6 +22,9 @@ jobs:
       - run:
           name: Audit
           command: npm audit --audit-level=low --production
+      - run:
```

Comment:
I was actually wondering if the lint step would be a required part of the build.  Like you can't build unless linting passes.  So there'd be an `npm build: '<lint command> <build command>'`.  If it makes sense to get linting in sooner than later, LGTM.

Reply:
Yeah, that's what we'd do. Linting and testing happen before the build command.